caas/kubernetes/provider/operator_test.go

```diff
@@ -621,6 +621,138 @@ func (s *K8sBrokerSuite) TestEnsureOperatorUpdate(c *gc.C) {
 	}
 }
 
+func (s *K8sBrokerSuite) TestEnsureOperatorNoStorage(c *gc.C) {
```

Comment:
This test is TestEnsureOperatorNoStorageKeepsPVC

Need another to ensure it handles no existing PVC and doesn't create one.

Reply:
yeah, just added a test for that. without a pvc, gets deployed as a deployment